packages/synapse-react-client/src/components/CardContainer/CardContainer.tsx

```diff
@@ -108,7 +108,7 @@ export function CardContainer(props: CardContainerProps) {
     schema[element.name] = index
   })
   const showViewMoreButton = (applyInitialLimit || hasNextPage) && (
-    <Box display="flex" justifyContent="flex-end">
+    <Box display="flex" justifyContent="flex-start">
```

Comment:
Left align the "View More" button so that its position is consistent when a max width is applied to Observation cards

Reply:
Good idea